### ISSUE TITLE
Add a mode to select a whole output

### ISSUE THREAD

**Issue by emersion** [emersion/slurp#18]

**Comment by Eluminae** [comment]
A mode or a middle/right click could be awesome.

**Comment by emersion** [comment]
I expect this feature to be superseded by https://github.com/emersion/slurp/pull/28

**Comment by emersion** [comment]
Fixed in #28

```
swaymsg -t get_outputs | jq -r '.[] | select(.active) | .rect | "\(.x),\(.y) \(.width)x\(.height)"' | slurp
```
